x-pack/platform/plugins/shared/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_home.tsx

```diff
@@ -67,7 +68,23 @@ export const ActionsConnectorsHome: React.FunctionComponent<RouteComponentProps<
     setIsLoadingActions(true);
     try {
       const actionsResponse = await loadAllActions({ http });
-      setActions(actionsResponse);
+      const filteredActionsResponse = await actionsResponse.reduce<Promise<ActionConnector[]>>(
+        async (result, connector) => {
+          if (
+            connector.actionTypeId !== '.inference' ||
+            (connector.actionTypeId === '.inference' &&
+              // @ts-ignore property 'config' does not exist on type ActionConnector
```

Comment:
Due to the time sensitive nature of this change, I did not update types as I wanted to keep code churn to a minimum. I can create a follow up to get rid of this ignore.

Reply:
I think it will be better to filter on the server side instead of the client making http calls for each .inference connector.

Reply:
it 100% should be on the server, otherwise there's too many places where this can be used and it's A) a lot of work to find/update them all, B) we risk missing something

Reply:
Also I agree with Yuliia, making this a blocking HTTP call per connector is not the right choice (at all).

Reply:
+1 on this.

Reply:
PR with the server-side fix is up https://github.com/elastic/kibana/pull/217641